Examples/JS/flow/BoardType.js

```diff
@@ -0,0 +1,23 @@
+//  @flow
+//
+//  BoardType.js
+//  Autogenerated by plank
+//
+//  DO NOT EDIT - EDITS WILL BE OVERWRITTEN
+//  @generated
+//
+
+import type { PlankDate, PlankURI } from './runtime.flow.js';
+import type { ImageType } from './ImageType.js';
+
+export type BoardType = $Shape<{|
+    +name: ?string,
```

Comment:
update: I think there has to be, since a lot of our JS code depends on `id` being a field on pins, boards, users, etc.